apps/opik-frontend/src/components/shared/DataTableCells/FeedbackScoreCell.tsx

```diff
@@ -13,10 +13,21 @@ import {
   getIsMultiValueFeedbackScore,
 } from "@/lib/feedback-scores";
 import FeedbackScoreCellValue from "./FeedbackScoreCellValue";
+import useTraceFeedbackScoreSetMutation from "@/api/traces/useTraceFeedbackScoreSetMutation";
+import { BaseTraceData } from "@/types/traces";
+import { useLoggedInUserName } from "@/store/AppStore";
+import useTraceFeedbackScoreDeleteMutation from "@/api/traces/useTraceFeedbackScoreDeleteMutation";
+import { USER_FEEDBACK_NAME } from "@/constants/shared";
 
 const FeedbackScoreCell = (context: CellContext<unknown, unknown>) => {
   const feedbackScore = context.getValue() as TraceFeedbackScore | undefined;
   const reason = feedbackScore?.reason;
+  const row = context.row.original as BaseTraceData;
+
+  const currentUserName = useLoggedInUserName() ?? "admin";
```

Comment:
do we use this admin user name anywhere? (is this for open source versions? )

Reply:
Yes, it's for local development (The assumed user)... otherwise, we don't have a user name for logic of users check, and since we support multi users editing 👍 👎 in user feedback, we need a userId for the mapping of **feedbackScore** per user:

feedbackScore?.value_by_author?.[currentUserName]?.value

To obtain **Personal user feedback**

Reply:
For other places, like experiments, we might need to change to `useLoggedInUserNameOrOpenSourceDefaultUser` from `useLoggedInUserName`, as is mentioned in the comment